cmd/frontend/internal/context/resolvers/context.go

```diff
@@ -407,8 +420,9 @@ func (r *Resolver) rerank(ctx context.Context, args graphqlbackend.RankContextAr
 func (r *Resolver) fetchZoekt(ctx context.Context, query string, repo *types.Repo) ([]graphqlbackend.RetrieverContextItemResolver, []error, error) {
 	var res []graphqlbackend.RetrieverContextItemResolver
 	fileChunks, err := r.contextClient.GetCodyContext(ctx, codycontext.GetContextArgs{
-		Repos: []types.RepoIDName{{ID: repo.ID, Name: repo.Name}},
-		Query: query,
+		Repos:        []types.RepoIDName{{ID: repo.ID, Name: repo.Name}},
+		FilePatterns: nil, // Not suppported in ChatContext
```

Comment:
Tiny suggestion as I was confused here
```suggestion
		FilePatterns: nil, // Not supported in ChatContext GraphQL API
```